plugin/src/main/java/net/dzikoysk/funnyguilds/feature/command/user/InviteCommand.java

```diff
@@ -42,14 +55,49 @@ public void execute(@CanManage User deputy, Guild guild, String[] args) {
         when(args.length < 1, this.messages.generalNoNickGiven);
         when(guild.getMembers().size() >= this.config.maxMembersInGuild, formatter.format(this.messages.inviteAmount));
 
-        User invitedUser = UserValidation.requireUserByName(args[0]);
+        if (args[0].equals(this.config.inviteCommandAllArgument)) {
+
+            double range;
+
+            if (args.length >= 2) {
+                range = Option.attempt(NumberFormatException.class, () -> Double.parseDouble(args[1])).orThrow(() -> new ValidationException(this.messages.inviteAllArgumentIsNotNumber));
+            }
+            else {
+                range = this.config.inviteCommandAllMaxRange;
+            }
+
+            when(range > this.config.inviteCommandAllMaxRange, FunnyFormatter.format(this.messages.inviteRangeToBig, "{MAX_RANGE}", this.config.inviteCommandAllMaxRange));
+
+            List<Player> nearbyPlayers = Bukkit.getServer().getOnlinePlayers().stream()
+                    .filter(player -> range >= player.getLocation().distanceSquared(sender.getLocation()))
+                    .filter(player -> player != sender)
```

Comment:
Porównujesz z dist squared, ale range to chyba po prostu odległość - więc chyba to nie będzie poprawne, nie?

Reply:
No... to zobaczyłem. Dzięki za info zmienie w next commicie